src/genie/libs/parser/iosxr/tests/test_show_pim.py

```diff
@@ -541,5 +541,80 @@ def test_show_pim_vrf_default_ipv6_rpf_summary_golden2(self):
         self.assertEqual(parsed_output,self.golden_parsed_output2)
 
 
+# ==========================================================================
+# Unittest for 'show pim topology summary'
+# ==========================================================================
+class test_show_pim_topology_summary(unittest.TestCase):
+    """ Unit test for show pim topology summary. """
+
+    empty_output = {'execute.return_value': ''}
+
+    golden_output_1 = {'execute.return_value': """
+        RP/0/RSP0/CPU0:PE5#show pim topology summary 
+        Tue Jan 26 04:06:02.148 UTC
+
+        PIM Topology Summary for VRF default
+          No. of group ranges = 5 (Active group ranges = 4)
+          No. of (*,G) routes = 1
+          No. of (S,G) routes = 200
+          No. of (S,G)RPT routes = 0
+        """}
+
+    golden_parsed_output_1 = {
+        'vrf':
+            {'default':
+                 {'active_group_ranges': 4,
+                  'no_group_ranges': 5,
+                  'no_sg_routes': 200,
+                  'no_sg_rpt_routes': 0,
+                  'no_stg_routes': 1
```

Comment:
we did below for `(*, G)` in different parser before. so better to have same as `no_g_routes`.
https://github.com/CiscoTestAutomation/genieparser/blob/master/src/genie/libs/parser/iosxr/show_mfib.py#L60
```
    {'vrf':
        {'all':
            {'no_g_routes': 20,
             'no_sg_routes': 176,
             'num_vrfs': 4
            }
        }
    }

Reply:
Hi @tahigash , yep just pushed it with the changes, thanks!

Reply:
confirmed the change